{
  "CVE Code": "CVE-2022-31313",
  "Published": "2022-06-08T19:13:13",
  "Updated": "2024-08-03T07:19:05.545Z",
  "Description": "api-res-py package in PyPI 0.1 is vulnerable to a code execution backdoor in the request package.",
  "ProblemTypes": [
    "n/a"
  ],
  "Affected": [
    {
      "vendor": "n/a",
      "product": "n/a",
      "versions": [
        "n/a"
      ]
    }
  ],
  "Reference": [
    "http://pypi.doubanio.com/simple/request",
    "https://pypi.org/project/api-res-py/",
    "https://github.com/rakeshrkz7/as_api_res/issues/1"
  ]
}